libc-test/semver/TODO-linux.txt

```diff
@@ -31,6 +31,7 @@ SO_ATTACH_REUSEPORT_EBPF
 SO_BINDTOIFINDEX
 SO_BPF_EXTENSIONS
 SO_BSDCOMPAT
+SO_BUSY_POLL_BUDGET
```

Comment:
Could you instead add these to `linux-gnu.txt` or `linux-gnu-x86_64.txt` instead?

Since there is something left to do, maybe leave a `// FIXME(musl)` in the `.rs` file rather than using the TODO semver.

Reply:
Good catch, will do!

Reply:
I think it needs to go in one of the `linux-gnu*` files rather than `linux-{arch}` since these aren't yet available on musl

Reply:
I added to the x86 one, but it should be fine for everything except musl, but it doesn't seem to have such a file